test/install_script/nvm_check_global_modules

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+
+cleanup () {
+  rm -rf "$npm_config_prefix/lib" >/dev/null 2>&1
+  unset npm_config_prefix
+
+  rm -f npm
+  PATH="$ORIGINAL_PATH"
+
+  unset -f setup cleanup die skip
+  unset message ORIGINAL_PATH
+}
+die () { echo $@ ; cleanup ; exit 1; }
+
+NVM_ENV=testing . ../../install.sh
+
+setup () {
+  ORIGINAL_PATH="$PATH"
+
+  npm_config_prefix="$(pwd)"
+  export npm_config_prefix
+  mkdir -p "$npm_config_prefix/lib"
+}
+
+
+setup
+
+npm install -g nop >/dev/null
+message=$(nvm_check_global_modules)
+[ ! -z "$message" ] || die "nvm_check_global_modules should have printed a notice when npm had global modules installed"
```

Comment:
the opposite of `! -z` is `-n`.

It may also be helpful here to run it first and do `>/dev/null 2>&1` and assert that the output (stderr) is empty (`-z`).